src/components/behandlerdialog/skrivtilbehandler/SkrivTilBehandlerSkjema.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useState } from "react";
+import { SkjemaFeiloppsummering } from "@/components/SkjemaFeiloppsummering";
+import { FlexRow } from "@/components/Layout";
+import { Button } from "@navikt/ds-react";
+import { Form } from "react-final-form";
+import { VelgBehandler } from "@/components/behandlerdialog/skrivtilbehandler/VelgBehandler";
+import { BehandlerDTO } from "@/data/behandler/BehandlerDTO";
+import styled from "styled-components";
+import { MeldingTekstfelt } from "@/components/behandlerdialog/skrivtilbehandler/MeldingTekstfelt";
+import {
+  behandlerRefValidationErrors,
+  validerSkjemaTekster,
+} from "@/utils/valideringUtils";
+import { useFeilUtbedret } from "@/hooks/useFeilUtbedret";
+
+const texts = {
+  knappTekst: "Send til behandler",
+};
+
+const validationTexts = {
+  missingMeldingTekst: "Innholdet i meldingen er tomt", // TODO: se nærmere på teksten
+};
+
+const SkrivTilBehandlerFormWrapper = styled.div`
+  margin-top: 1.5em;
+`;
+
+const StyledForm = styled.form`
+  > * {
+    &:not(:last-child) {
+      margin-bottom: 1.5em;
+    }
+  }
+`;
+
+interface SkrivTilBehandlerSkjemaValues {
+  behandlerRef: string;
+  meldingTekst: string;
+}
+
+interface SkrivTilBehandlerSkjemaTekster {
+  meldingTekst: string;
+}
+
+type SkrivTilBehandlerSkjemaFeil = Partial<
```

Comment:
Tror ikke du trenger en egen type for denne (og den blir vel uansett samme som `Partial<SkrivTilBehandlerSkjemaValues>` siden du plukker alle propertiene derfra)

Reply:
Allright, det var nok veldig inspirert av annen kode, men ser ut til at det kan fjernes ja 🙌🏼